Umbraco-Cloud/Troubleshooting/Deployments/Connection-Forcibly/index.md

```diff
@@ -0,0 +1,18 @@
+---
+versionFrom: 7.0.0
+---
+
+# "An existing connection was forcibly closed by the remote host" error
+
+Sometimes you will get an error that looks like this:
+
+    System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
+    System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.
+    System.Net.WebException: The underlying connection was closed: An unexpected error occurred on a send.
+    System.Net.Http.HttpRequestException: An error occurred while sending the request.
```

Comment:
Instead of indenting these log entries, how about wrapping them in a code block instead?
And then usu XML or json as the syntax highlighting? 

You can see more about how to do that here: https://our.umbraco.com/documentation/Contribute/Code-samples/

Reply:
Fixed in this: https://github.com/umbraco/UmbracoDocs/pull/2783/commits/157c019479672476e2917fb203c40597bfe4196a